daml-lf/governance.rst

```diff
@@ -5,109 +5,144 @@ DAML-LF Governance process
+  semantics of existing features, or update the serialization format.
+  Changes to the minor component cannot break backward compatibility,
+  and operate on the same major version of the serialization format in
+  a backward compatible way. For historical reason this scheme
+  starts at `1.6`.
+
+* DAML-LF values, nodes, and transactions are versioned using a common
+  one component version scheme, called transaction version scheme.
+  For historical reason this scheme starts at `10`.
+  
+* DAML-LF Language, values, nodes, and transactions are evolved
+  together without ever breaking backwards compatibility.
+  
+* DAML-LF values, nodes, and transactions, and language are involved
+  together, meaning the introduction of a new language version implies
+  the introduction of a new transaction version, and reciprocally.
+
+* A specification is provides for each major version language, and A
+  common specification document are provided for values, nodes,
+  transactions.  Moreover, a one-line summary of each version must be
+  present in the ``daml-lf-X.proto`` and ``transaction.proto`` files.
+
+* Every change to DAML-LF values, nodes, transactions or languages
+  *must* be checked for governance rules conformance and approved must
+  be approved by at least two persons among Rémy Haemmerle, Martin
+  Huschenbett and Sofia Faro.
+
+  These checks are for governance and versioning only; the substance
+  of DAML-LF language changes should be made in consultation with the
   Language team.  Proposed changes can be filed as tickets under
-  Milestone "DAML-LF Spec & Validation", labels "component/daml-lf" and
-  "discussion".
-
-* Every ledger implementation must declare what DAML-LF value,
-  transaction, and language versions it supports. This information will
-  be most likely tied to what DAML-LF engine the ledger is running, but
-  additional constraints external to the DAML-LF engine can be put in
-  place, if necessary. Note that a running ledger can never drop support
-  for value, transaction, or language versions, since it must be able to
-  replay every transaction ever run on the ledger. However the ledger is
-  free to refuse to accept new code running with older DAML-LF versions
-  while still maintaining support for existing code internally.
-
-* Client libraries and other tooling mostly care about DAML-LF values
-  and DAML-LF serializable types. Deciding how these components handle
-  versions is not part of the DAML-LF governance progress. For
-  example, a certain version of the DAML-LF Scala codegen might
-  support a specified set of DAML-LF versions -- much like ledger
-  implementations.
-
-Values
-------
+  labels "component/daml-lf" and "discussion".
+
+Language
+--------
+
+Some version bumps might not change the ``.proto`` at all, but just
+add / change semantics in the specification. For example, the string
+format for decimal numbers in DAML-LF values might be made more
```

Comment:
This sounds like we're breaking backward compatibility. Maybe we need to be more specific.

Reply:
Again it is just paragraph that I moved around. 
I am not sure how we can be more specific. 

``
For example, the string format for decimal numbers in DAML-LF values might be made more lenient in newer version  than in the original `1.10`.
``

Reply:
Ok, if it was here before, then let's leave it as is.